### PR TITLE
chore: rename ag to agriculture for clarity

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR changes all "Ag" abbreviations to "Agriculture" to improve clarity & better adhere to conventions.

### CODE REVIEW

**Comment by nissimsan** [comment]
Excellent!

**Review by nissimsan — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/AgricultureInspectionReport.yml [hunk 1]
```diff
@@ -260,10 +260,10 @@ example: |-
         "phoneNumber": "555-895-1661",
         "faxNumber": "555-497-2527"
       },
-      "AgPackage": [
+      "agriculturePackage": [
```
Comment (brownoxford) [inline docs/openapi/components/schemas/common/AgricultureInspectionReport.yml:263]:
```suggestion
      "AgriculturePackage": [
```

**Comment by BenjaminMoe** [comment]
Attributes should be camel case. Classes should be title case.

**Comment by BenjaminMoe** [comment]
Handle in another PR.

**Pull request merged**